src/StreamStats.jl

```diff
@@ -1,7 +1,7 @@
 module StreamStats
     import StatsBase, Distributions
 
-    export update!, state, nobs
+    export update!, state, nobs, rand, replicates!
```

Comment:
We can't export rand: that's in Base. We can only extend it by doing something like `Base.rand(foo::MyNewType) = 1`.